### ISSUE TITLE
Initial Infected Keep & Can Use Their Hands

### ISSUE THREAD

**Issue by PursuitInAshes** [space-wizards/space-station-14#19890]
## Description

When a player is initial infected and turn (I believe both through the action and forceturned, but I have only seen through the action first hand), their hands aren't removed (And held items aren't dropped). This allows them to use firearms, tools, etc. while being a zombie.

**Reproduction**
(From what I have seen.)

1. Be Initial Infected (Roundstart or mid-round event both work from my understanding_
2. Hold Something in your hands and use the action to turn (Alternatively be forcedturned due to time.)
3. Item will remain in hand and is useable.

Note: I don't have time to test and confirm (At the moment.), but from what I have been told, the hands are fully useable like standard, and just aren't removed as they should be. Leading to initial infected being capable using firearms, stun batons, etc.

**Screenshots**
(If someone is able to test and take a screenshot, I can add it here if need be.)

**Additional context**

I've bumped into this issue first hand only once, a warden forced turned with a stun baton in hand, and continued to hold it after being turned.

**Comment by hord-brayden** [comment]
I have been meaning to pile this on - I was a rat king, got infected, and was still able to use my abilities and operate as normal but with the added zombie benefits. Able to consume food, miasma, make servants, etc. unsure if that's related to this, or I can make a new issue.
